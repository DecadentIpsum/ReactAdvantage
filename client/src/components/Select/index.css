.select {
  color: #1E1E1E;
  background: #F5F5F3;
  margin-bottom: 10px; }
  .select .ui-multiselect-panel .ui-multiselect-close {
    color: #1E1E1E; }
  .select .ui-widget-content {
    color: #1E1E1E;
    background: #FFF; }
