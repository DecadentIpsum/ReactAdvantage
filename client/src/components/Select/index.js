import React, { Component } from 'react';
import {MultiSelect as PrimeMultiSelect} from 'primereact/components/multiselect/MultiSelect';

import './index.css'

export default class MultiSelect extends Component {
    render() {
        return (
            <div >
                <PrimeMultiSelect className="select" {...this.props} />
            </div>
        );
    }
}
