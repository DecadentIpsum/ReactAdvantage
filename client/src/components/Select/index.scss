.select {
    color: #1E1E1E;
    background: #F5F5F3;

    .ui-multiselect-panel .ui-multiselect-close {
        color: #1E1E1E;
    }

    .ui-widget-content {
        color: #1E1E1E;
        background: #FFF;
    }
}