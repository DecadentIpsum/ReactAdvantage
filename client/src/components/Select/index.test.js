import React from 'react';
import Enzyme, { shallow } from 'enzyme';

import Select from './index';

it('renders correctly', () => {
    const component = shallow(<Select />);

    expect(component).toMatchSnapshot();
});