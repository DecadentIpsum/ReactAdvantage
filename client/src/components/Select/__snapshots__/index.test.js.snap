// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MultiSelect />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <MultiSelect
        appendTo={null}
        className="select"
        defaultLabel="Choose"
        disabled={false}
        filter={false}
        id={null}
        itemTemplate={null}
        key={null}
        onChange={null}
        optionLabel={null}
        options={null}
        scrollHeight="200px"
        style={null}
        value={null}
/>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "appendTo": null,
        "className": "select",
        "defaultLabel": "Choose",
        "disabled": false,
        "filter": false,
        "id": null,
        "itemTemplate": null,
        "key": null,
        "onChange": null,
        "optionLabel": null,
        "options": null,
        "scrollHeight": "200px",
        "style": null,
        "value": null,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <MultiSelect
          appendTo={null}
          className="select"
          defaultLabel="Choose"
          disabled={false}
          filter={false}
          id={null}
          itemTemplate={null}
          key={null}
          onChange={null}
          optionLabel={null}
          options={null}
          scrollHeight="200px"
          style={null}
          value={null}
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "appendTo": null,
          "className": "select",
          "defaultLabel": "Choose",
          "disabled": false,
          "filter": false,
          "id": null,
          "itemTemplate": null,
          "key": null,
          "onChange": null,
          "optionLabel": null,
          "options": null,
          "scrollHeight": "200px",
          "style": null,
          "value": null,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
